src/components/Chip/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ButtonBridge, {IButtonBridge} from '../ButtonBridge';
+import useStyles from './styles';
+
+const Chip = (
+  props: Pick<
+    IButtonBridge,
```

Comment:
if you pick just selected props from the IButtonBridge it means that you can't pass props from the PressableProps  that the original Button interface extends, right?

Reply:
yeah you can only pass the picked ones

Reply:
I think in this case it should be changed to "Omit" unwanted props, so you can still use the basic props from pressable. Not sure if this makes for some use cases in react-native, but its very useful in web IMO

Reply:
I have picked the ones that are only allowed in Chip for limiting the props

Reply:
Right, but point is that if I use "Button" i can also pass anything from "Pressable" native props, and if I use a Chip/Tag I can't. On web when using Button, Chip or Tag you have some limitations on "our" props, but you still can pass any prop from the native HTMLAttributes<HTMLButtonElement> interface. I think we should keep this consistent - might be worth discussing it today